eng/Version.Details.xml

```diff
@@ -3,13 +3,13 @@
   <ProductDependencies>
   </ProductDependencies>
   <ToolsetDependencies>
-    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="1.0.0-beta.19405.1">
+    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="1.0.0-beta.19517.3">
```

Comment:
Is there any chance there's a non-beta version of Arcade available somewhere?

Reply:
I'll have a snoop and see what I can find. Wouldn't surprise me if there wasn't though.

I remember they only made a 1.0.0 of SourceLink because people thought it hadn't been "officially" released yet. I think stuff they don't ship just seems to be a forever increasing beta.

Reply:
Looking at dotnet/aspnetcore, they're on a beta of version 5 now:

https://github.com/dotnet/aspnetcore/blob/602f4678c1e764e4e4c4bcdf26a5f83abe39ecdb/eng/Version.Details.xml#L415

Reply:
Roger. If `1.0.0-beta.19517` works fine with 3.1, we can't wait before moving to 5.0.